bayesian/api_v1.py

```diff
@@ -246,6 +246,7 @@ class ComponentAnalyses(Resource):
     @staticmethod
     def get(ecosystem, package, version):
         """Handle the GET REST API call."""
+        secy_vendor = request.headers.get('secy_vendor', None)
```

Comment:
Either use "security_vendor" or use "sec_vendor"

Reply:
`secy` is an acronym for security. Either way, I am okay to change it to `security`